<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectListWindowBaseClass</class>
 <widget class="QMainWindow" name="ObjectListWindowBaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="SkTableView" name="table">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="searchEdit">
        <property name="toolTip">
         <string>Enter search text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clearButton">
        <property name="toolTip">
         <string>Clear search text</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>388</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuObject">
    <property name="title">
     <string>&amp;Object</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string extracomment="The mnemonic is referenced in ObjectListWindow&lt;T&gt;, make sure to update it when this value is changed.">&amp;Window</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuWindow"/>
   <addaction name="menuObject"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string extracomment="New">Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string extracomment="Delete">Ctrl+D</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="shortcut">
    <string extracomment="Edit">Ctrl+E</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string extracomment="Refresh">Ctrl+R</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string extracomment="Close">Ctrl+W</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SkTableView</class>
   <extends>QTableView</extends>
   <header>src/gui/widgets/SkTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ObjectListWindowBaseClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>692</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
